components/list/list-item-drag-handle.js

```diff
@@ -38,6 +38,8 @@ class ListItemDragHandle extends LocalizeCoreElement(LitElement) {
 		return {
 			disabled: { type: Boolean, reflect: true },
 			text: { type: String },
+			size: { type: Number },
```

Comment:
It's a bit strange to have a `size` associated with the handle of each item

Reply:
Agreed, but for other reasons mentioned in previous comment.